src/chapters/exceptions/exercises.rst

```diff
@@ -0,0 +1,29 @@
+Exercises
+=========
+
+To get started, fork and clone this `repo <https://github.com/LaunchCodeEducation/csharp-web-dev-lsn9exceptions>`_ from Github. 
+
+Divide By Zero!
+---------------
+
+Complete a function called ``Divide()`` in ``Program.cs``.
+``Divide()`` takes in two parameters: ``x`` and ``y``.
+
+Your function should return the result of ``x/y``.
+
+However, if ``y`` is zero you should throw an appropriate error message.
+Try to use an ``ArgumentOutOfRangeException`` and put your ``try/catch`` block in ``Main()`` to test out your error-handling skills!
+
+Test Student Labs
+-----------------
+
+After mentioning to Professor Jackson that you would like to get some more practice with exceptions, she offered to let you write some grading software!
```

Comment:
who the heck is this? can you provide more context here if you're telling a story. is this meant to be a theme through the whole page or specifically this task?

in the same vein, is there a connection between these two methods? if there is none, consider placing these in separate files and wrapper classes. we've spent the last several lessons introducing object oriented concepts and it makes sense to practice what we preach.

Reply:
Changed intro and now the two are better connected!